### ISSUE TITLE
breeding?

### ISSUE THREAD

**Issue by m3t4f1v3** [frikinjay/mob-stacker#10]
how does breeding work with this mod installed? as of now I have to separate and assimilate all of them to breed

**Comment by frikinjay** [comment]
Yes, that is the intended way to breed mobs, having breeding give the stacked number of resulting mobs would somewhat defeat the purpose of the mod.
